applications/crust-mobile-wallet.md

```diff
@@ -94,16 +94,16 @@ I have development experience, including front-end development, back-end develop
 | 5. | Test Android| TestFlight beta testing|
 | 6. | Go live | Submit app to App Store And Google Play|
 
-### Milestone 4: Staking 
+### Milestone 4: Staking && Storage app
 * **Estimated Duration:** 2 month
 * **FTE:**  1
 * **Costs:** 7,000 USD
 
 | Number | Deliverable | Specification |
 | -----: | ----------- | ------------- |
 | 1. | Staking| Adding staking app|
-| 2. | Governance| Adding governance ??|
-
+| 2. | Storage apps| integrate the storage function of Crust|
```

Comment:
We prefer if you have a prototype/design at first, so we can setup a meeting and talk about it

Reply:
Give me more time try to setup design and let you know

Reply:
Cool, you can reach out to me through email: dean@crust.network,  if you have any questions.